### ISSUE TITLE
[1.12.2] Living Enchantment doesn't work if in your Off-Hand. Also... Living Enchantment in your Main-Hand affects non-enchanted items in your Off-Hand.

### ISSUE THREAD

**Issue by SonicX8000** [Clownvin/Living-Enchantment#11]
Minecraft: 1.12.2
Forge: 14.23.5.2847

Mods: 2
jei_1.12.2-4.15.0.291
living_enchantment-3.1.9

-----
Living Enchantment does nothing if you have it in your Off-Hand Slot. Also... It seems that if you have a weapon with the Living Enchantment in your Main-Hand and you use an item in your off-hand, such as a Bow with no Living Enchantment. The Damage is increased despite the Bow not having Living. Here are some tests using the Damage Indicators Mod to keep track of the damage.

With the way I have the configs set up. The Level cap is 20 and the damage boost is double the weapon's damage. For example... a Diamond Sword has 7 Damage and at Lv 20 via Living... it'll have 14 Damage.

***Test 1***
Main-Hand: Nothing
Off-Hand: Bow, Normal Arrows.
Mob: Iron Golem

I fire 5 arrows at the Golem and damage inflicted is...
9, 6, 8, 10, 7

'Notes'
Normal stuff here.

-----

***Test 2***
Main-Hand: Diamond Sword enchanted with Living, Lv 20, 14 DMG
Off-Hand: Bow, Normal Arrows
Mob: Iron Golem

I fire 5 arrows at the Golem and damage inflicted is...
18, 18, 18, 14, 20

'Notes'
Despite the Bow's not being enchanted with 'Living'. The damage is increased due to having a Living Enchantment in my Main-Hand.

-----

***Test 3***
Main-Hand: Bow, Normal Arrows
Off-Hand: Diamond Sword enchanted with Living, Lv 20, 14 DMG.
Mob: Iron Golem

I fire 5 arrows at the Golem and damage inflicted is...
9, 11, 11, 9, 10

'Notes'
Looks to be the same as Test 1 where the damage dealt is normal.

-----

***Test 4***
Main-Hand: Diamond Sword enchanted with Living, Lv 20, 14 DMG
Off-Hand: Bow enchanted with Living, Normal Arrows, Lv 20, ??? DMG
Mob: Iron Golem

I fire 5 arrows at the Golem and damage inflicted is...
18, 18, 16, 18, 18

'Notes'
Looks to be just like Test 2 so at least they're not stacking with each other.

-----

***Test 5***
Main-Hand: Nothing
Off-Hand: Bow enchanted with Living, Normal Arrows, Lv 20, ??? DMG
Mob: Iron Golem

I fire 5 arrows at the Golem and damage inflicted is...
8, 10, 10, 7, 10

'Notes'
Despite the Bow being enchanted with Living. If using it from your Off-Hand... the damage isn't increased at all. Swapping it to my Main-Hand and the damage is increased just like with Tests 2 & 4.

==========

In short, Living Enchantment will only apply it's damage boost via the Main-Hand only. It also effects non-enchanted items in your Off-Hand if you have a Living Enchantment in your Main-Hand. This can kinda be broken as one can having Living in their Main-Hand and use... say a Tinker's Shuriken, Longbow, etc. in their off-hand which can lead to some high damage being done.
